.github/scripts/ubuntu-20.04/download_build.sh

```diff
@@ -0,0 +1,22 @@
+# Download installers for compiled dependencies
+
+mkdir -p ${DOWNLOAD_PATH}
+
+## Thrift 0.16
+
+wget -O ${DOWNLOAD_PATH}/thrift-0.16.0.tar.gz "http://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=thrift/0.16.0/thrift-0.16.0.tar.gz"
+
+# Calculate hash of dependencies for Github Cache Action
+
+dependencies_to_hash=("thrift-0.16.0.tar.gz")
+
+concatenated_data=""
+for file in "${dependencies_to_hash[@]}"; do
+    concatenated_data="${concatenated_data}$(echo -n "$file"; cat "${DOWNLOAD_PATH}/${file}")"
```

Comment:
Concatenating the complete binary content of all dependencies maybe shouldn't be done, instead just concatenate hashes. (And probably do a second hashing after that, but that could be optional.)

Reply:
Concatenating only the hashes indeed should be enough. Fixed.